resources/aws/peering.go

```diff
@@ -145,3 +145,7 @@ func (v *VPCPeeringConnection) GetID() (string, error) {
 
 	return *vpcPeeringConnection.VpcPeeringConnectionId, nil
 }
+
+func (v VPCPeeringConnection) FindExisting() (*ec2.VpcPeeringConnection, error) {
```

Comment:
I needed this to get some `VPCPeeringConnection` properties, this was called in the same package before.

Reply:
Good to move this to the resource.